JamJar/ConcertPageViewController.swift

```diff
@@ -0,0 +1,226 @@
+//
+//  ConcertPageViewController.swift
+//  JamJar
+//
+//  Created by Mark Koh on 3/12/16.
+//  Copyright © 2016 JamJar. All rights reserved.
+//
+
+import UIKit
+import SCLAlertView
+
+class ConcertPageViewController: BaseViewController, UITableViewDelegate, UITableViewDataSource, UISearchBarDelegate {
+    
+    @IBOutlet weak var tableView: UITableView!
+    @IBOutlet weak var artistImageView: UIImageView!
+    @IBOutlet weak var artistLabel: UILabel!
+    @IBOutlet weak var dateLabel: UILabel!
+    @IBOutlet weak var venueLabel: UILabel!
+    
+    var concert: Concert? = nil
+    var videos: [Video] = [Video]()
+    var myVideos: [Video] = [Video]()
+    var individualVideos: [Video] = [Video]()
+    var jamjars: [JamJarGraph] = [JamJarGraph]()
+    
+    var showJamJars: Bool = true
+    var showMyVideos: Bool = true
+    var showIndividualVideos: Bool = true
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        // Register the reusable video cell
+        self.tableView.registerNib(UINib(nibName: "VideoCell", bundle: nil), forCellReuseIdentifier: "VideoCell")
+        self.tableView.registerNib(UINib(nibName: "JamJarCell", bundle: nil), forCellReuseIdentifier: "JamJarCell")
+        self.tableView.registerNib(UINib(nibName: "JamJarHeaderCell", bundle: nil), forCellReuseIdentifier: "JamJarHeaderCell")
+        
+        // Set JamJars
+        ConcertService.getJamJars((self.concert?.id)!) {
```

Comment:
Let's take this out of the `viewDidLoad` function and put it into its own function, then add the function to a "pull to refresh" target.